calcom/cal.com
# Broken Integration Alarm System

## What does this PR do?



This PR alerts users when an integration is broken.

When a video integration is broken
![image](https://user-images.githubusercontent.com/65426560/175120141-5e729e5a-cddb-4a26-98bf-790704b94e68.png)

When there's an error with the calendar
![image](https://user-images.githubusercontent.com/65426560/175120203-e8084ff0-721a-41ee-852d-8fa3a0895662.png)


Fixes #2944



**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Broken video credential
	- In your .env file go to the Daily credentials and delete the api key
	- Edit an event type and set the location to Cal Video
	- Create a booking, you should get the broken integration email for video
- Broken calendar credential
	- Connect a Google calendar
	- In the DB go to the google calendar credential, under key delete some characters from the access token
	- Create a booking, you should get the broken integration email for calendars
## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
